apps/masterbots.ai/components/icons/closeIcon.tsx

```
@@ -0,0 +1,18 @@
+const CloseIcon = (props: React.ComponentPropsWithoutRef<'svg'>) => (
+  <svg width={19} height={18} viewBox="0 0 19 18" fill="none" {...props}>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)